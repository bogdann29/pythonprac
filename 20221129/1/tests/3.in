class A(metaclass=dump):
    def __init__(self, val=5):
        self.val = val * 2

class B(A):
    def print(self, b):
        print(b ** 2)


a = B(val=7)
a.print(1)