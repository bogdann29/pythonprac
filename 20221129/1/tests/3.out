__init__: (), {'val': 7}
print: (1,), {}
1
